### PR TITLE
Storybook

### PR DESCRIPTION
**Pull request by sidforhbc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "lint": "npm run lint:css && npm run lint:js",
     "lint:css": "stylelint src/**/*.scss",
     "lint:js": "eslint src/ --ext .js --ext .jsx",
+    "storybook": "start-storybook -s ./ -p 9001 -c .storybook",
```
Comment (track0x1) [inline package.json]: Minor, but what do you think about naming the storybook folder without a leading `.`?

File: src/actions/MiniBagActionCreators.js [hunk 2]
```diff
@@ -44,7 +44,9 @@ function getServiceMiniBag(dispatch) {
     // private mode is throwing `DOM Exception 22`
     try {
       sessionStorage.setItem(LOCAL_MINI_BAG_KEY, JSON.stringify(json.data));
-        } catch (e) {} // eslint-disable-line
+    } catch (e) {
+      return false;
```
Comment (track0x1) [inline src/actions/MiniBagActionCreators.js]: Do you know if the intention is to imply that a "DOM Exception 22" still means a successful action?

Reason I say that is, if we're catching the error in here and just returning false, I don't think the promise's `catch` is going to catch this unless we throw an error.

File: src/components/HeaderUtilsBag/HeaderUtilsBag.jsx [hunk 3]
```diff
@@ -54,7 +52,7 @@ const HeaderUtilsBag = React.createClass({
       <a
         href={this.props.shoppingBagLink}
         className="header-utils__link header-utils__link--bag"
-        onClick={this.onClickHandler}>
+        onClick={this.handleClick}>
```
Comment (track0x1) [inline src/components/HeaderUtilsBag/HeaderUtilsBag.jsx]: So good.

File: src/utilities/RewardsTooltipUtils.jsx [hunk 4]
```diff
@@ -8,7 +8,9 @@ function cacheTooltipSupportInfo(res) {
   // private mode is throwing `DOM Exception 22`
   try {
     localStorage.setItem(LOCAL_REWARDS_HEADER_TOOLTIP_KEY, JSON.stringify(res));
-    } catch (e) {} // eslint-disable-line
+  } catch (e) {
+    return false;
```
Comment (track0x1) [inline src/utilities/RewardsTooltipUtils.jsx]: See similar comment I mentioned [here](https://github.com/hbc-tech/header-ui/pull/3/files/b90828d55adb1590b3c8e781235e612c9f6b29a8#diff-3ec7026998e2d74a5f2d9baefededddf).

File: src/actions/MiniBagActionCreators.js (hunk 2 above)
Comment (sidforhbc) [reply src/actions/MiniBagActionCreators.js]: Good question and honestly don't know. The mini bag is not in the greatest shape but I'll make sure to add a note here to double-check this during the next pass

File: .storybook/webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        loader: 'json-loader'
+      }
+    ]
+  },
+  resolve: {
+    extensions: ['.js', '.json', '.jsx'],
+    alias: {
+      hammer: path.resolve(__dirname, '../src/shims/Hammer.js'),
+      modernizr: path.resolve(__dirname, '../src/shims/Modernizr.js'),
+      react: path.resolve(__dirname, '../node_modules/react')
```
[the remark this answers is not in the document]
Comment (sidforhbc) [reply .storybook/webpack.config.js]: To quote our famous comrade: copypasta

**Pull request merged**